dd-java-agent/instrumentation/jax-rs-annotations-2/src/test/groovy/JaxRsFilterTest.groovy

```diff
@@ -0,0 +1,179 @@
+import datadog.trace.agent.test.AgentTestRunner
+import datadog.trace.api.DDSpanTypes
+import datadog.trace.api.DDTags
+import datadog.trace.instrumentation.api.Tags
+import io.dropwizard.testing.junit.ResourceTestRule
+import javax.ws.rs.client.Entity
+import javax.ws.rs.container.ContainerRequestContext
+import javax.ws.rs.container.ContainerRequestFilter
+import javax.ws.rs.container.PreMatching
+import javax.ws.rs.core.MediaType
+import javax.ws.rs.core.Response
+import javax.ws.rs.ext.Provider
```

Comment:
It looks like import order was changed during merge conflict resolution. Do you have correct import layout for groovy (https://github.com/open-telemetry/opentelemetry-auto-instr-java/blob/master/CONTRIBUTING.md#intellij-idea)?

Reply:
I do, and that is why it was reformatted when I updated other parts of the file.  It seems the original author doesn't have it configured.  Unless there's an easy way to validate it like we do with Java, I don't worry about it too much unless it starts thrashing a lot.

Reply:
Looks like I lied... my settings were not the default.  I've fixed my settings, but not going to worry about reformatting now.... eventual consistency 😉